### ISSUE TITLE
selenium-wire is no longer maintained

### ISSUE THREAD

**Issue by fabaff** [dirkjanm/ROADtools#101]
In January 2024 `selenium-wire` was [archived](https://github.com/wkeeling/selenium-wire/commit/da1b675fe2cc6dae2e3bb959c1ce95eb1c41830b). `roadtx` [depends](https://github.com/dirkjanm/ROADtools/blob/master/roadtx/setup.py#L24) on `selenium-wire`.

`selenium-wire` is not pinning `blinker` which leads to issues (`ModuleNotFoundError: No module named 'blinker._saferef'`).

For most distribution the pinning of `blinker` to > 1.8.0 will no help but for PyPI it would. In a best case scenario `selenium-wire` would be dropped for `roadtx`.

Thanks

**Comment by dirkjanm** [comment]
I agree, but at the moment there is no obvious solution that can be used as a drop-in replacement. Either roadtx would use it's own "fork" or lite clone of selenium-wire, or a rewrite of the intercepting element would be needed.

**Comment by dirkjanm** [comment]
roadtx now uses a fork with some fixes from https://github.com/dirkjanm/selenium-wire-roadtx/, which removes the deprecated stuff and makes it work without pinning blinker
